share/completions/apk.fish

```diff
@@ -6,8 +6,8 @@ complete -f -c apk -n "__fish_seen_subcommand_from del fix version manifest" -a
 
 # Global options
 complete -f -c apk -s h -l help -d "Show help"
-complete -x -c apk -s p -l root -d "Install packages to DIR"
-complete -x -c apk -s X -l repository -d "Use packages from REPO"
+complete -F -c apk -s p -l root -r -d "Install packages to DIR"
+complete -F -c apk -s X -l repository -r -d "Use packages from REPO"
```

Comment:
are you sure that REPO is a filename?

Reply:
It can point to local directory, so the path completion should be available imo.
Example of `apk add`:
```sh
apk add -X ~/packages/main <package_name>
```

Example of `/etc/apk/repositories`:
```
#/media/cdrom/apks

@aports-m /home/pj/packages/main
@aports-c /home/pj/packages/community
@aports-t /home/pj/packages/testing

https://dl-cdn.alpinelinux.org/alpine/v3.15/main
https://dl-cdn.alpinelinux.org/alpine/v3.15/community
```